Sources/TuistGraph/Models/Plugins.swift

```diff
@@ -1,20 +1,30 @@
 import Foundation
+import TSCBasic
 
 /// A model which contains all loaded plugin representations.
 public struct Plugins: Equatable {
     /// List of the loaded custom helper plugins.
     public let projectDescriptionHelpers: [ProjectDescriptionHelpersPlugin]
 
+    /// List of paths to template definitions.
+    public let templateDirectories: [AbsolutePath]
+
     /// Creates a `Plugins`.
     ///
     /// - Parameters:
     ///     - projectDescriptionHelpers: List of the loaded helper plugins.
+    ///     - templates: List of templates loaded from plugins.
```

Comment:
nit: this does not correspond to the current parameter name

I'd also rename it to `templateDirectories` to be the same as the property (or the other way around, I don't really any preferences regarding that)